### ISSUE TITLE
Remove Custom Cursor (For Now)

### ISSUE THREAD

**Issue by stiefeljackal** [Yellow-Dog-Man/go.resonite.com#77]
The custom cursor for the website should be removed for now until there are personal settings in `go.resonite.com` to disable it. The reason why I bring this up is due to accessibility reasons as overriding the cursor appearance preferences defined on the system will change the user's cursor size and appearance. Here is a video demonstrating this:

https://github.com/user-attachments/assets/a077b554-e01b-4ce9-8609-c3d3ed3e2bd1

The user may need to use a cursor that is larger and/or is specialized in appearance due to low vision issues, and having the cursor size and appearance abruptly change compromises that. Although I am for giving a website some character, accessibility should come first. As mentioned, the custom cursor can come back when there is a way to customize the personal preferences (perhaps after the architecture change), but it should be removed for now.

**Comment by Stellanora64** [comment]
The custom cursor does just feel pretty jarring to me. If it was a standalone application there is a bit more precedent for the application changing the cursor, but I can't think of any other examples of web utilities like go.resonite using custom cursors (or even websites in general).

**Comment by ProbablePrime** [comment]
The fixes in #78 improve the experience here. 

I agree with the concerns here, but i also don't want to remove the cursors right now. Lets see if there's additional feedback here from users and that might help us make the decision after 78 is deployed.
